KnockKnock-iOS/Scenes/Cells/FeedListCell.swift

```diff
@@ -0,0 +1,302 @@
+//
+//  FeedListCell.swift
+//  KnockKnock-iOS
+//
+//  Created by Daye on 2022/06/22.
+//
+
+import UIKit
+
+import KKDSKit
+
+class FeedListCell: BaseCollectionViewCell {
```

Comment:
```suggestion
final class FeedListCell: BaseCollectionViewCell {
```
final 을 작성하는게 처음에는 익숙하지 않죠 ㅎㅎ 확인 부탁드립니다.